Update set-up-phone-system-voicemail.md (#4901)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/set-up-phone-system-voicemail.md

> [!NOTE]
> Cloud Voicemail supports depositing voicemail messages only in an Exchange mailbox and doesn't support any third-party email systems. 

> ## Cloud-only environments: Set up Cloud Voicemail for Cloud Psystem Users

Reviewer: ```suggestion
## Cloud-only environments: Set up Cloud Voicemail for Cloud Phone System users
```

6. To enable Voicemail features such as customizing greetings, and visual voicemail in Skype for Business clients, connectivity from Office 365 to the Exchange server mailbox via Exchange Web Services is required. To enable this connectivity you must configure the new Exchange Oauth authentication protocol described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/library/dn594521(v=exchg.150).aspx), or run the Exchange Hybrid Wizard from Exchange 2013 CU5 or greater. Additionally, you must configure integration and Oauth between Skype for Business Online and Exchange server described in [Configure Integration and OAuth between Skype for Business Online and Exchange Server](https://docs.microsoft.com/skypeforbusiness/deploy/integrate-with-exchange-server/oauth-with-online-and-on-premises). 

## Set up Cloud Voicemail for Skype for Business Server Users

The following information is about configuring Cloud Voicemail to work with users who are online for Exchange and on-premises for Skype for Business. 
  

> 1. You may need to purchase Exchange Online licenses to the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).

Reviewer: ```suggestion
1. You may need to purchase Exchange Online licenses for the users in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```

> 1. You may need to purchase Exchange Online licenses to the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).

Reviewer: ```suggestion
1. You may need to purchase Exchange Online licenses for the people in your business. See [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
```

> 2. [Assign or remove licenses for Office 365 for business](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc) the Exchange Online licenses to the people in your business.

Reviewer: ```suggestion
2. To assign the Exchange Online licenses to users in your organization, please see [Add users individually or in bulk](https://support.office.com/article/997596b5-4173-4627-b915-36abac6786dc).
```

    
3. Support for voicemail transcription has been added as of March 2017 and is enabled by default for all organizations and users. You can disable transcription for your organization by using Windows PowerShell and following the steps below.

> 4. To configure Skype for Business server users for Cloud Voicemail please see [Plan Cloud Voicemail service for on-premises users](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-cloud-voicemail)

Reviewer: ```suggestion
4. To configure Skype for Business server users for Cloud Voicemail, please see [Plan Cloud Voicemail service for on-premises users](https://docs.microsoft.com/en-us/skypeforbusiness/hybrid/plan-cloud-voicemail).
```
